test/interpreter_functional/plugins/kbn_tp_run_pipeline/public/np_ready/app/components/main.tsx

```diff
@@ -31,7 +31,7 @@ declare global {
       expressions: string,
       context?: ExpressionValue,
       initialContext?: ExpressionValue
-    ) => ReturnType<ExpressionDataHandler['getData']>;
+    ) => any;
```

Comment:
Why not use `ExecutionContract['getData']`?

Reply:
Because expression result can be anything, thus correct type here is `any`.